### PR TITLE
docs(select): add playgrounds for toggle icon customization

### PR DESCRIPTION
**Pull request by amandaejohnston**
Adds two playgrounds:
- Usage for the new `toggleIcon` and `toggleIconWhenOpen` props added in https://github.com/ionic-team/ionic-framework/pull/27648
- Example CSS for customizing icon flip behavior, in lieu of a new prop as detailed [here](https://github.com/ionic-team/ionic-framework-design-documents/blob/main/projects/ionic-framework/components/select/0003-custom-icon-on-open.md#modifying-flip-behavior)

Dev build for testing: `7.0.15-dev.11687278023.161b97d8` (note that you'll need to add this to see the custom icons in the playground iframes)

Shortcut: https://ionic-docs-git-fw-2134-ionic1.vercel.app/docs/api/select#custom-toggle-icons

### CODE REVIEW

File: static/usage/v7/select/customization/custom-toggle-icons/index.md
Comment (liamdebeasi) [inline static/usage/v7/select/customization/custom-toggle-icons/index.md:10]:
```suggestion
  size="400px"
```

The "interface popover" playground uses a height of 400px otherwise the popover appears above the button

File: static/usage/v7/select/customization/icon-flip-behavior/index.md
Comment (liamdebeasi) [inline static/usage/v7/select/customization/icon-flip-behavior/index.md:15]:
```suggestion
  size="400px"
```

The "interface popover" playground uses a height of 400px otherwise the popover appears above the button

File: static/usage/v7/select/customization/custom-toggle-icons/react.md
Comment (liamdebeasi) [inline static/usage/v7/select/customization/custom-toggle-icons/react.md:11]: Ionic React needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { add, remove } from 'ionicons/icons';

...

toggleIcon={add}
```

File: static/usage/v7/select/customization/custom-toggle-icons/vue.md
Comment (liamdebeasi) [inline static/usage/v7/select/customization/custom-toggle-icons/vue.md:8]: Ionic Vue needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { add, remove } from 'ionicons/icons';

...

:toggle-icon="add"
```

File: static/usage/v7/select/customization/icon-flip-behavior/react/main_tsx.md
Comment (liamdebeasi) [inline static/usage/v7/select/customization/icon-flip-behavior/react/main_tsx.md:13]: Ionic React needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { caretDownSharp } from 'ionicons/icons';

...

toggleIcon={caretDownSharp}
```

File: static/usage/v7/select/customization/icon-flip-behavior/vue.md
Comment (liamdebeasi) [inline static/usage/v7/select/customization/icon-flip-behavior/vue.md:7]: Ionic Vue needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { caretDownSharp } from 'ionicons/icons';

...

:toggle-icon="caretDownSharp"
```

File: static/usage/v7/select/customization/icon-flip-behavior/vue.md
Comment (liamdebeasi) [inline static/usage/v7/select/customization/icon-flip-behavior/vue.md:20]: Ionic Vue needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { caretDownSharp } from 'ionicons/icons';

...

:toggle-icon="caretDownSharp"
```

File: static/usage/v7/select/customization/icon-flip-behavior/react/main_tsx.md
Comment (liamdebeasi) [inline static/usage/v7/select/customization/icon-flip-behavior/react/main_tsx.md:26]: Ionic React needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { caretDownSharp } from 'ionicons/icons';

...

toggleIcon={caretDownSharp}
```

**Review by liamdebeasi — CHANGES_REQUESTED** [review]

**Review by liamdebeasi — APPROVED** [review]
Changes look great! Let's hold off on merging until the bikeshedding for the feature is done. I'm going to try and get that across the finish line after standup today.

**Comment by liamdebeasi** [comment]
As per the updates on https://github.com/ionic-team/ionic-framework/pull/27648, we'll need to change references to `toggleIconWhenOpen` to `expandedIcon`. Once completed this is good to merge.

**Comment by amandaejohnston** [comment]
References have been updated, just need to create a new dev build.

**Pull request merged**
